@types/interfaces.d.ts

```diff
@@ -0,0 +1,14 @@
+export interface MenuBarItem {
+  text?: string;
+  component?: string | HTMLElement;
+  disabled?: boolean;
+  children?: SubMenuItem[];
+}
+
+export interface SubMenuItem {
```

Comment:
I was thinking that maybe we can make this extend `ContextMenuItem` interface, but decided to keep them separate for now, so as we keep considering `vaadin-context-menu` an implementation detail.